client/src/index.jsx

```diff
@@ -16,7 +16,7 @@ ReactDOM.render(
   <React.StrictMode>
     <Router>
       <Switch>
-        <Route path='/recipe/:recipeId'>
+        <Route path='/recipeId'>
```

Comment:
Is this a typo? I can navigate to the Recipe Detail Page, but I think we want to use `recipeId`.

Reply:
Not a typo, was working on an offline version so I dont make 10000 API calls in 10 seconds by accident. When we go to production we can change it all to be referencing the actual ID.